cpp/tests/copying/concatenate_tests.cpp

```diff
@@ -404,7 +404,7 @@ TEST_F(OverflowTest, OverflowTest)
       static_cast<cudf::size_type>(static_cast<uint32_t>(512) * 1024 * 1024);
 
     // struct
-    std::vector<std::unique_ptr<column>> children;
+    std::vector<std::unique_ptr<cudf::column>> children;
```

Comment:
Did we remove a `using namespace cudf` or `namespace cudf {` somewhere that I'm missing in this file? There seem to be a large number of changes unrelated to the column API change, and I don't see why they would even be necessary based on the focus of the PR. Perhaps those changes should be split into a separate PR?

Reply:
The offending lines removed are 46-47 above.
```
using column = cudf::column;
using column_view = cudf::column_view;
```
I could not just let that be.

Reply:
I did not realize how big a change that turned out to be. I can certainly do that change in a separate PR.